tutor/resources/styles/components/top-nav-bar/center-controls.scss

```diff
@@ -58,6 +58,7 @@
   .annotation-summary-toggle {
     background-color: transparent;
     border: none;
+    outline: none;
     .i {
```

Comment:
I think we do want the focus outline style on the summary toggle, or did I miss a request to remove it?